vue/store.js

```diff
@@ -36,6 +36,11 @@ export const store = new Vuex.Store({
         currentFrame: null,
         error: null,
         previous: null,
+        detailsTitle: null,

```

Comment:
@gcandal We need to move this RIPE White specific store information out of here. Proper inheritance or composition should be used for the specific details of RIPE White. Also, @gcandal update https://github.com/ripe-tech/ripe-white/issues/93 implementation part to talk about the update of the store.